src/OpenLoco/src/GameCommands/Track/RemoveSignal.cpp

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+#include "RemoveSignal.h"
+#include "Economy/Economy.h"
+#include "Map/SignalElement.h"
+#include "Map/TileManager.h"
+#include "Map/Track/TrackData.h"
+#include "Map/TrackElement.h"
+#include "Objects/TrainSignalObject.h"
+#include "Vehicles/Vehicle.h"
+#include "ViewportManager.h"
+
+namespace OpenLoco::GameCommands
+{
+    static World::TrackElement* getElTrackAt(const SignalRemovalArgs& args, const World::Pos3 pos, const uint8_t index)
```

Comment:
```suggestion
    static World::TrackElement* getElTrackAt(const SignalRemovalArgs& args, const World::Pos3 pos, const uint8_t sequenceIndex)
```
I think index is a bit too vague here.